packages/hooks/src/start.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+import { spawn } from 'child_process';
+import path from 'path';
+import fs from 'fs';
+
+const { main: pkgJSONMain } = JSON.parse(fs.readFileSync('./package.json', 'utf-8'));
+
+/**
+ * Start hook implementation that verifies and runs an app in Socket Mode.
+ * @param {string} cwd - The current working directory of the project.
+ * @param {string | undefined} customPath - An optional path for the app.
+ */
+(function _(cwd, customPath) {
+  // TODO - Format so that its less miss-able in output
+  console.log('Preparing local run in developer mode (Socket Mode)'); // eslint-disable-line no-console
```

Comment:
Something about the phrase "local run" is niggling at me. That makes a lot of sense in the hosted vs. local paradigm, but less so in Bolt world. Thoughts on this? Just me?

Reply:
I'm kinda leaning towards removing the first two logs in this hook entirely 😳 Or possibly included these in some message-boundary and displayed only in CLI debug logs?

I agree that it's a strange output. Especially in the Bolt context. Ignoring `DEBUG` logs from the Bolt app itself, we're left with this output which feels noisy to me...

```txt
$ slack run -w largespeck
Updating local app install for "largespek"
✨ somename of largespek
Preparing local run in developer mode (Socket Mode)
/Users/somename/programming/samples/bolt-js-starter-template/app.js
[INFO]  socket-mode:SocketModeClient:0 Going to establish a new connection to Slack ...
⚡️ Bolt app is running! ⚡️
[INFO]  socket-mode:SocketModeClient:0 Now connected to Slack
```

It's a really good question though, since we're advising `slack run` is only used for development and something else for deployed apps.

Maybe this line changes to something like "Preparing development app in Socket Mode"?

Reply:
I vote remove them then. Keep it clean and tidy!

Reply:
Update: These are logs are removed. Clean and tidy just feels better. Also, the error message for a missing `app.js` (or whatever's specified in the `package.json` is clear IMO)

<details>
<summary>This is the error message</summary>

```sh
$ pwd
/private/tmp/hooks
$ slack run
? Choose a local environment devrelsandbox T038J6TH5PF A06FQE9PVK4
Updating local app install for "DevRel Sandbox"
✨ ethan.zimbelman of DevRel Sandbox
node:internal/modules/cjs/loader:1051
  throw err;
  ^

Error: Cannot find module '/private/tmp/hooks/apppppppppppp.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1048:15)
    at Module._load (node:internal/modules/cjs/loader:901:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:83:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.8.0
Local run exited with code 1
```